.gitignore

```diff
@@ -48,3 +48,6 @@ public/__sitemaps/
 
 #A spot for developers to put miscellaneous scripts.
 /personal_bin/*
+
+# db name & credentials for local OH microsite import database
+/bin/oh_microsite_export/local_database*
```

Comment:
This works fine, but this could be `/bin/oh_microsite_export/local_database/*`, files under that dir, instead of possibly other dirs starting with same name. This doesn't need to be fixed for us to merge, just good to know it's a possibility.